cap/01-categorie.tex

```diff
@@ -1,9 +1,162 @@
+  \item Una classe $\ctC_1$ i cui elementi chiamiamo \emph{morfismi}, di solito indicati con lettere latine minuscole: $f$, $g$, $h$,\dots
+  
+  Queste classi sono fornite delle seguenti strutture. 
+  
+  \item Ad ogni morfismo corrispondono due oggetti $\dom{f}$, $\cod{f}$ chiamati \emph{dominio} e \emph{codominio}. Se $f$ ha dominio $X$ e codominio $Y$, scriviamo $f:X\to Y$, o in forma diagrammatica, $X \xrightarrow{f} Y$.
+  \item Ogni oggetto $X$ ha un particolare morfismo $\id_X:X\to X$ chiamato \emph{identità}.
+  \item Dati tre oggetti $X,Y,Z$ e morfismi $f:X\to Y$ e $g:Y\to Z$ (si noti che $\cod{f}=\dom{g}$) esiste un particolare morfismo $g\circ f:X\to Z$ chiamato \emph{composizione di $f$ e $g$}. Graficamente:
+  $$
+  \begin{tikzcd}
+   X \ar{r}{f} \ar[bend right=20]{rr}[swap]{g\circ f} & Y \ar{r}{g} & Z .
+  \end{tikzcd}
+  $$
+  (Si noti che $\dom{g\circ f}=\dom{f}$ e $\cod{g\circ f}=\cod{g}$.)
+  
+  Queste strutture devono soddisfare le seguenti proprietà.
+  
+  \item \emph{Unitalità}: per ogni morfismo $f:X\to Y$, le composizioni $f\circ\id_X$ e $\id_Y\circ f$ sono uguali ad $f$.
+  \item \emph{Associatività}: dati oggetti $X,Y,Z,A$ e morfismi $f:X\to Y$, $g:Y\to Z$ e $h:Z\to A$, le composizioni $h\circ (g\circ f)$ e $(h\circ g)\circ f$ sono uguali.
+ \end{itemize}
+\end{definition}
+
+Dati due oggetti $X$ e $Y$, indichiamo con $\Hom{\ctC}(X,Y)$ la classe di morfismi da $X$ a $Y$. Altri autori usano altre notazioni, come $\mathrm{Hom}(X,Y)$ o $\mathrm{Hom}_\ctC(X,Y)$.
+
+
+Quando le classi $\ctC_0$ e $\ctC_1$ sono insiemi, chiamamo la categoria $\ctC$ \emph{piccola}. Più in generale, $\ctC$ si dice \emph{localmente piccola} se dati ogni due oggetti $X$ e $Y$, la classe $\Hom{\ctC}(X,Y)$ di morfismi da $X$ a $Y$ è un insieme. 
+
+\begin{examples} 
+ Vediamo alcuni esempi classici di categorie. 
+ \begin{itemize}
+  \item Come accennato sopra, gli spazi vettoriali su $\ictR$ e le mappe lineari formano una categoria, che chiamiamo $\ctVect_\ictR$. Più in generale possiamo definire la categoria $\ctVect_\ictF$, per un campo $\ictF$, in maniera analoga.
+  \item Gli insiemi e le funzioni formano una categoria, che chiamiamo $\ctSet$.
+  \item I gruppi e gli omomorfismi di gruppo formano la categoria $\ctGrp$.
+  \item I gruppi abeliani e gli omomorfismi di gruppo formano la sottocategoria $\ctAb\subseteq\ctGrp$. (Definiremo le sottocategorie a breve.)
+  \item Gli spazi topologici e le funzioni continue formano la categoria $\ctTop$. 
+ \end{itemize}
+Spesso un ambito della matematica si occupa di una o più categorie in particolare. Per esempio l'algebra lineare si occupa principalmente delle categorie $\ctVect_\ictF$.
+\end{examples}
+
+Le categorie negli esempi sopra si possono vedere come categorie dove gli oggetti sono insiemi con strutture ulteriori (per esempio, strutture di gruppo), e i morfismi sono funzioni che preservano queste strutture. Non tutte le categorie sono di questo tipo, come vediamo nei prossimi esempi. 
```

Comment:
Chiuderei questo in un remark (e rifraserei un po': c'è una ripetizione di "categorie" a pochissima distanza)